wiki/People/Global_Moderation_Team/es.md

```diff
@@ -21,45 +21,52 @@ Miembros del equipo
 
 **Ten en cuenta: Todos los miembros hablan inglés a menos que se diga lo contrario.** - [Team Page](http://osu.ppy.sh/g/4)
 
-| Nombres | Idiomas | Enfoque |
+| Nombre | Idioma | Enfoque |
 | ----- | --------- | ------------- |
-| [deadbeat](https://osu.ppy.sh/u/128370) | | Creación de osu!media |
-| [Dntm8kmeeatu](https://osu.ppy.sh/u/5428812) | | Soporte técnico |
-| [ErunamoJAZZ](https://osu.ppy.sh/u/1869764) | Español | Moderacion del chat |
-| [Flanster](https://osu.ppy.sh/u/447818) | Búlgaro | osu!idol hosting, Moderacion del chat |
-| [Galkan](https://osu.ppy.sh/u/169570) | Polaco | Moderacion del chat |
-| [ghm12](https://osu.ppy.sh/u/2594229) | Portugués | Moderacion del chat |
-| [IamKwaN](https://osu.ppy.sh/u/1856463) | Cantonés, Chino | Moderacion del chat |
-| [juankristal](https://osu.ppy.sh/u/443656) | Español | Moderacion del chat, Torneos |
-| [Kobold84](https://osu.ppy.sh/u/3227533) | Ruso | Moderacion del chat |
-| [KSHR](https://osu.ppy.sh/u/409957) | Japonés | Moderacion del chat |
-| [Kurai](https://osu.ppy.sh/u/77089) | Francés, Ruso | Moderacion del chat |
-| [Marcin](https://osu.ppy.sh/u/722665) | Polaco | Moderación del foro, Desarrollo |
-| [MrSergio](https://osu.ppy.sh/u/2581696) | Italiano | Moderacion del chat |
-| [Nathanael](https://osu.ppy.sh/u/2295078) | Filipino | Moderación del foro, Moderacion del chat |
-| [Neil Watts](https://osu.ppy.sh/u/3048059) | Francés | Moderacion del chat |
-| [Nishiji](https://osu.ppy.sh/u/167629) | Turco | Moderacion del chat |
+| [-kevincela-](https://osu.ppy.sh/u/266596) | Italiano | |
+| [[ Another ]](https://osu.ppy.sh/u/3416573) | Húngaro | Chat moderation |
```

Comment:
The entire file is using the English titles on the **Enfoque (Area of Focus)** column.

Reply:
It is actually fixed.